src/icons/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@nypl/design-system-icons",
+  "version": "0.0.0",
+  "description": "Icons for the Design System",
+  "main": "dist/img/sprite/sprite.svg",
+  "scripts": {
+    "prepare": "npm run build",
+    "build": "node src/scripts/build-icon.js",
+    "test": "echo \"No Icon tests\""
```

Comment:
1st test:  The validation would mostly be a test for `spriter.compile`, which isn't our code and probably isn't necessary to test.  Could be good to test that our source svgs are valid, though, and have the test serve as documentation on what happens when you pass in a bad svg.

2nd test:  Is this to make sure that designers can't drop in bad (for us) svgs?  I think this could be super useful, and I also think it's a big enough task that we should either extract into a ticket or into a backlog task so as not to overload this one.

Reply:
1st test: I agree, I was hoping we would run that on our `src/` directory, since that's the one we control. This could also be where we would catch uncompressed icons?
2nd test: I'm mostly thinking of testing bad svgs. It's not uncommon when you preprocess them for something funky to happen and nothing shows up.

Let's definitely dump this into the backlog—I don't want to make this ticket larger!

Reply:
re: first test - I started writing these and it feels like testing `svg-sprite` so we shouldn't do that.

Reply:
I've added a [ticket to the backlog](https://github.com/NYPL/nypl-design-system/projects/2#card-29721175) to think through how we want to test icons. Let's not block on it here.